powerdnsadmin/assets.py

```diff
@@ -58,6 +59,8 @@ def concat(self, out, hunks, **kw):
                  'node_modules/jtimeout/src/jTimeout.js',
                  'node_modules/jquery.quicksearch/src/jquery.quicksearch.js',
                  'custom/js/custom.js',
+                 'node_modules/bootstrap-datepicker/js/bootstrap-datepicker.js',
```

Comment:
Could you remove this line? Will merge subsequently.

Reply:
Thanks for the remark. 
I added the change you suggested.

Reply:
Thanks! Merged.